src/components/FeaturedCardsCarousel/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Carousel, { ResponsiveType } from 'react-multi-carousel';
+
+import 'react-multi-carousel/lib/styles.css';
+
+import { FeaturedCards } from '../../lib/types';
+import FeaturedCard from '../FeaturedCard';
+
+const responsive: ResponsiveType = {
```

Comment:
For the moment and for the amount of FeaturedCards we have, I would leave this as follows:

**superLargeDesktop:** 3 items
**largeDesktop:** 3 items
**desktop:** 2 items
**tablet:** 1 item
**mobile:** 1 item